setup.sh

```diff
@@ -22,7 +22,10 @@ echo "pip installing requirements from requirements file in target directory"
 pip install -r ../requirements.txt -t .
 
 # Prepares the deployment package
+
+#Need to confirm that zip installed.. setup runs without errors otherwise 
 echo "Zipping package"
+sudo apt-install zip
```

Comment:
@suramrit is this command going to break things cross-platform?

Might be better to let people know in the readme that they need `zip` and point them to installation instructions?

Reply:
Yeah. Ill just leave the comment there.